### PR TITLE
Geeks3D.MSIKombustor version 4.1.15.0

### PR DESCRIPTION
**Pull request by Amoenus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36709)

### CODE REVIEW

File: manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Geeks3D.MSIKombustor4
```
Comment (KaranKad) [inline manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Geeks3D.MSIKombustor.4
```

File: manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Geeks3D.MSIKombustor4
```
Comment (KaranKad) [inline manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.installer.yaml:4]:
```suggestion
PackageIdentifier: Geeks3D.MSIKombustor.4
```

File: manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Geeks3D.MSIKombustor4
```
Comment (KaranKad) [inline manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.yaml:4]:
```suggestion
PackageIdentifier: Geeks3D.MSIKombustor.4
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Like https://github.com/microsoft/winget-pkgs/pull/26659, it would be preferable if the download came from https://www.geeks3d.com/furmark/kombustor/downloads/.

image
image

**Comment by KaranKad** [comment]
Edit folder structure using github.dev

**Comment by KaranKad** [comment]
@ItzLevvie Their download link redirects to that link
image

**Comment by OfficialEsco** [comment]
Just checked their other softwares, MSIKombustor is the only download link that redirect to another site
Every other software that is on their servers just loop back around like FurMark
https://github.com/microsoft/winget-pkgs/pull/26659#issuecomment-912943352

**Comment by Amoenus** [comment]
@KaranKad @ItzLevvie are we positive we want to use Geeks3D.MSIKombustor.4?
It seems a bit odd-looking to me. I was going back and forth between Geeks3D.MSIKombustor and Geeks3D.MSIKombustor4

I went for the second option because of an installer:
image

It always mentions MSI Kombustor 4 even in the default install path
I usually rely on the `winget list` as a tie-breaker or a hint on how to determine new identifier. Sadly this package gives worst case scenario with just a guid and unknown version...
image

**Comment by OfficialEsco** [comment]
Example of this splitting: https://github.com/microsoft/winget-pkgs/tree/master/manifests/c/Corsair/iCUE

**Comment by Amoenus** [comment]
This is a similar issue I am having with picking the right identifier for LEGOEducation.WeDo
image

image

This one is even weirder as it is `WeDo 2.0` version `1.9.x`

File: manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Geeks3D.MSIKombustor4
+PackageVersion: 4.1.15.0
+PackageLocale: en-US
+Publisher: MSI / Geeks3D
+PackageName: MSI Kombustor 4
```
Comment (ItzLevvie) [inline manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.locale.en-US.yaml:8]:
```suggestion
PackageName: MSI Kombustor
```

Comment (ItzLevvie) [reply manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.locale.en-US.yaml:8]: > It seems a bit odd-looking to me. I was going back and forth between Geeks3D.MSIKombustor and Geeks3D.MSIKombustor4

Knowing what we do with other installers, I would probably vouch for the former.

If Control Panel showed `MSI Kombustor 4 version 4.1.15.0` then we could probably go for `Geeks3D.MSIKombustor.4` with `MSI Kombustor 4` as PackageName.

At the moment, Control Panel shows `MSI Kombustor 4.1.15.0` so if we remove the version, it'll be `MSI Kombustor`

If you want to add older versions of MSI Kombustor, then you could split it up into `Geeks3D.MSIKombustor.4`; `Geeks3D.MSIKombustor.3`, etc.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
> This one is even weirder as it is WeDo 2.0 version 1.9.x

I think for that `LEGOEducation.WeDo` would be fine.

File: manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.installer.yaml (hunk 2 above)
Comment (Amoenus) [reply manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.installer.yaml:4]: Will go with Geeks3D.MSIKombustor

File: manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.yaml (hunk 3 above)
Comment (Amoenus) [reply manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.yaml:4]: Will go with Geeks3D.MSIKombustor

File: manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.locale.en-US.yaml (hunk 1 above)
Comment (Amoenus) [reply manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.locale.en-US.yaml:4]: Will go with Geeks3D.MSIKombustor

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**